libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -297,6 +305,10 @@ export class ChangeAction extends BaseRushAction {
     } catch (error) {
       throw new Error(`There was an error creating a change file: ${(error as Error).toString()}`);
     }
+    if (this._commitChangesFlagParameter.value) {
+      this._stageAndCommitGitChanges([this.rushConfiguration.changesFolder], 'Rush changes');
```

Comment:
nit:

I think this commit message should be configurable from `rush.json` too.
We can extend `gitPolicy` object with new field, like `changesCommitMessage`, we just need to make sure that it is distinguishable from `changeLogUpdateCommitMessage` that is already there

Reply:
Added